src/peliasDocGenerators.js

```diff
@@ -73,7 +73,11 @@ module.exports.create = function(hierarchy_finder) {
           wofDoc.addParent('macroregion', hierarchy_element.name, hierarchy_element.id.toString());
           break;
         case 'dependency':
-          wofDoc.addParent('dependency', hierarchy_element.name, hierarchy_element.id.toString());
+          if (hierarchy_element.abbreviation) {
```

Comment:
It will take improvements to the model code because it blows up gloriously on empty/null values due to the validation routines.